docs/2.0 EDID to Receiver Caps mapping.md

```diff
@@ -138,9 +141,7 @@ All sampling frequencies MUST go to `urn:x-nmos:cap:format:sample_rate` as `enum
 
 
 [RFC-2119]: https://tools.ietf.org/html/rfc2119 "Key words for use in RFCs to Indicate Requirement Levels"
-
+[IS-04]: https://specs.amwa.tv/is-04/ "AMWA IS-04 NMOS Discovery and Registration Specification (Stable)"
```

Comment:
Maybe we need to refer to it somewhere, because it's currently an orphaned link.

Reply:
it's used here https://github.com/AMWA-TV/nmos-edid-connection-management/pull/35/files#diff-529bd43592af1e685eb0f6f90948951d291dec2ba395423ec709090d04083ae4R7